README.md

```diff
@@ -7,51 +7,53 @@ This project was built using only built in web services (ie no NPM packages besi
 ## 🔑 Technical Highlights and Key APIs
 
 ### 💾 System File Access API
-- Save/Save As
-- Open
-- Limitations
-  - Privacy on the web (Losing the file handler on refresh)
-  - Requires user permission
-
-### ✒️ System Font API
-- Read fonts from users OS
-- Fall back on the default list
-- Limitations
-  - Are there stytles in the OS that aren't supported in CSS
-  - Requires user permission
+
+The System File Access API is a MUST inlcude for any sort of app that is going to want to read/write user files. For the Notepad PWA, we used this API to handle opening and saving files. Even though this API requires permission (which we will talk more about later), it is central to the apps functioning, so we will rely on the user accepting this API to have access to their file system in order for the app to work as expected.
+
+### ✒️ Local Font Access API
+
+The Notepad native application offers a lot of different font families and styles to select and use in the editor. Instead of manually porting the fonts over to the PWA, we can just use the Local Font Access API to grab all available fonts on the users operating system. In the spirit of PWA, this will allow the available fonts to  be different across different devices (ie Android will have different font selections compared to Windows).
+
+This API is only supported in chromium, so that limits where the PWA can be used, but fret not, if the API is unavailable on your OS, there should be a fallback list that the PWA will rely on.
+
+The Windows OS provides some fonts like "Wingdings 2" and some specific styles like "Sitka Heading" that do not necessarily map to fonts available, by default, on the Web. This is important to consider when using the Local Font Access API to be sure to set up fallbacks or handle cases where the font does not map.
+
+### User permission
+The above two API's both will require user permission. I consider this the double edge sword of the web. For developers, you have to consider the case where the user does not allow access for the necessary API. Sometimes the API is central to the app functioning, like the File System API in this PWA so if this were to be a _real_ app, we would probably make sure this was accepted before moving forward. Requesting permission to access certain information it gives the user full transparency and control over what the app is reaching into on their computer which can transfer to the user being more comfortable using a PWA.
+
+This is a difference between PWA and Native applications that can not be avoided. This should no be viewed as a strictly negative thing because it does indeed benefit the user.
```

Comment:
```suggestion
This is a difference between PWA and Native applications that cannot be avoided. This should not be viewed as a strictly negative thing because it does indeed benefit the user.
```